### PR TITLE
Implement sample to demonstrate MP-JWT feature

### PR DESCRIPTION
**Pull request by madawas**
This commit adds a sample service written using eclipse miroprofile 1.2 framework which is secured using mp-jwt 1.0.

Identity server can be used to generate a JWT token which can be used to invoke the endpoints of this service.

### CODE REVIEW

File: modules/samples/microprofile/microprofile-jwt/pom.xml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                <artifactId>maven-war-plugin</artifactId>
+                <configuration>
+                    <failOnMissingWebXml>false</failOnMissingWebXml>
+                    <packagingExcludes>pom.xml</packagingExcludes>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>net.wasdev.wlp.maven.plugins</groupId>
+                <artifactId>liberty-maven-plugin</artifactId>
+                <version>2.2</version>
```
Comment (pulasthi7) [inline modules/samples/microprofile/microprofile-jwt/pom.xml]: Move the versions to the plugin management section in root pom

File: modules/samples/microprofile/microprofile-jwt/pom.xml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+    <artifactId>microprofile-jwt</artifactId>
+
+    <packaging>war</packaging>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.eclipse.microprofile</groupId>
+            <artifactId>microprofile</artifactId>
+            <version>1.2</version>
```
Comment (pulasthi7) [inline modules/samples/microprofile/microprofile-jwt/pom.xml]: Add this to dependency management section in root pom and remove the version,type from here

File: pom.xml [hunk 3]
```diff
@@ -532,6 +532,11 @@
                     <artifactId>build-helper-maven-plugin</artifactId>
                     <version>3.0.0</version>
                 </plugin>
+                <plugin>
+                    <groupId>net.wasdev.wlp.maven.plugins</groupId>
+                    <artifactId>liberty-maven-plugin</artifactId>
+                    <version>2.2</version>
```
Comment (pulasthi7) [inline pom.xml]: Shall we define a property for this one also?

Comment (madawas) [reply pom.xml]: fixed with 7340ce6

File: modules/samples/microprofile/microprofile-jwt/pom.xml (hunk 2 above)
Comment (madawas) [reply modules/samples/microprofile/microprofile-jwt/pom.xml]: Fixed with a3ae797

File: modules/samples/microprofile/microprofile-jwt/pom.xml (hunk 1 above)
Comment (madawas) [reply modules/samples/microprofile/microprofile-jwt/pom.xml]: Fixed with a3ae797

**Pull request merged**